InitializationProtocol/ManagementProtocol.adoc

```diff
@@ -0,0 +1,135 @@
+OmniXtend Management Protocol Documentation
+===========================================
+:Author: Nathaniel Graff
+:Email: <nathaniel.graff@sifive.com>
+
+== Introduction
+
+The OmniXtend Management Protocol describes the process by which a cluster of
+OmniXtend Nodes is configured from cold boot by an OmniXtend Coordinator and
+transitioned to the next stage of the boot flow.
+
+=== Glossary
+
+OmniXtend Cluster::
+	A collection of machines, connected by Ethernet, which includes both OmniXtend Nodes, an OmniXtend Coordinator, and optionally network switches.
+OmniXtend Node::
+	One of several machines in an OmniXtend Cluster which implements the TileLink over Ethernet protocol (TLoE).
+OmniXtend Coordinator::
+	A single machine in the OmniXtend Cluster which performs the Coordinator role in the OmniXtend Management Protocol. The Coordinator may be any machine connected to the same Ethernet network capable of sending and receiving Ethernet packets so that the Coordinator role may be performed by any of a separate physical server, a programmable network switch, or an OmniXtend Node.
```

Comment:
Rename it to management server.

Reply:
Two hard problems in computer science (cache invalidation, naming things, and off by one errors) and this project encompasses all of them. 😅

Reply:
:)